cpp/daal/src/sycl/sum_reducer.cpp

```diff
@@ -47,13 +47,12 @@ services::Status buildProgram(ClKernelFactoryIface & kernelFactory, const TypeId
     return status;
 }
 
-services::Status sum_singlepass(ExecutionContextIface & context, ClKernelFactoryIface & kernelFactory, Layout vectorsLayout,
-                                const UniversalBuffer & vectors, uint32_t nVectors, uint32_t vectorSize, uint32_t workItemsPerGroup,
-                                SumReducer::Result & result)
+services::Status sum_singlepass(ExecutionContextIface & context, ClKernelFactoryIface & kernelFactory, const UniversalBuffer & vectors,
+                                uint32_t nVectors, uint32_t vectorSize, SumReducer::Result & result)
 {
     services::Status status;
 
-    auto sum_kernel = kernelFactory.getKernel("sum_singlepass", status);
+    auto sum_kernel = kernelFactory.getKernel("sum_singlesubgroup", status);
```

Comment:
Does this change affect the performance of other algorithms, except KMeans?

Reply:
Discussed offline - @amgrigoriev says that every row-major case where sum-reducer is used in the library shall speed up.